dapps/profile.vue/src/components/wallet/send/SendEve.ts

```diff
@@ -231,7 +224,7 @@ export default class SendEveComponent extends mixins(EvanComponent) {
   handleWindowResize() {
     this.windowWidth = window.innerWidth;
     if (this.windowWidth >= 1200) {
-      (<any>this).$store.state.uiState.swipePanel = 'sharing';
+      (<any> this).$store.state.uiState.swipePanel = 'sharing';
```

Comment:
no any needed ANY more :D

Reply:
or there should actually the commit on store be used: `this.$store.commit('toggleSidePanel', 'sharing');` to avoid reactivity issues ;)